routers/web/repo/setting_protected_branch.go

```diff
@@ -115,21 +117,20 @@ func SettingsProtectedBranch(c *context.Context) {
 	c.Data["whitelist_users"] = strings.Join(base.Int64sToStrings(rule.WhitelistUserIDs), ",")
 	c.Data["merge_whitelist_users"] = strings.Join(base.Int64sToStrings(rule.MergeWhitelistUserIDs), ",")
 	c.Data["approvals_whitelist_users"] = strings.Join(base.Int64sToStrings(rule.ApprovalsWhitelistUserIDs), ",")
+	c.Data["status_check_contexts"] = strings.Join(rule.StatusCheckContexts, "\n")
 	contexts, _ := git_model.FindRepoRecentCommitStatusContexts(c, c.Repo.Repository.ID, 7*24*time.Hour) // Find last week status check contexts
-	for _, ctx := range rule.StatusCheckContexts {
-		var found bool
-		for i := range contexts {
-			if contexts[i] == ctx {
-				found = true
-				break
+	c.Data["branch_status_check_contexts"] = contexts
+	c.Data["can_context_match"] = func(context string) bool {
```

Comment:
It's not needed, since frontend JS is necessary, it could do all the work.

Reply:
removed `can_context_match` in fb9d67d2403c8edea47c4ca82acc1fd25467958d